components/organisms/CollaborationRequestWrapper/collaboration-requests-wrapper.tsx

```diff
@@ -14,7 +23,55 @@ function formatPostDate(date: Date) {
   }
 }
 
-const CollaborationRequestsWrapper = () => {
+type EmailPreferenceType = {
```

Comment:
```suggestion
interface EmailPreferenceType {
```

I think for our propose it is enough the interface.